### PR TITLE
SDBELGA-262 [EVENT FILES] Save additional file information.

### PR DESCRIPTION
**Pull request by ride90**
In mongo:
Screenshot 2020-01-17 at 16 28 41

Production API response:
Screenshot 2020-01-17 at 16 38 51

### CODE REVIEW

File: server/planning/events/events_files.py [hunk 1]
```diff
@@ -45,7 +46,16 @@ class EventsFilesService(superdesk.Service):
...
             # save the media id to retrieve the file later
-            doc['filemeta'] = {'media_id': doc['media']}
+            _file = app.media.get(doc['media'])
+            doc['filemeta'] = {
+                'media_id': doc['media'],
+                'chunk_size': _file.chunk_size,
+                'content_type': _file.content_type,
+                'filename': _file.filename,
+                'length': _file.length,
+                'name': _file.name,
```
Comment (petrjasek) [inline server/planning/events/events_files.py]: isn't this same as filename?

File: server/planning/events/events_files.py [hunk 2]
```diff
@@ -45,7 +46,16 @@ class EventsFilesService(superdesk.Service):
     def on_create(self, docs):
         for doc in docs:
             # save the media id to retrieve the file later
-            doc['filemeta'] = {'media_id': doc['media']}
+            _file = app.media.get(doc['media'])
+            doc['filemeta'] = {
+                'media_id': doc['media'],
+                'chunk_size': _file.chunk_size,
```
Comment (petrjasek) [inline server/planning/events/events_files.py]: I think this is mongo specific, wouldn't expose it

File: server/planning/events/events_files.py [hunk 3]
```diff
@@ -45,7 +46,16 @@ class EventsFilesService(superdesk.Service):
...
+                'content_type': _file.content_type,
+                'filename': _file.filename,
+                'length': _file.length,
+                'name': _file.name,
+                'upload_date': _file.upload_date
```
Comment (petrjasek) [inline server/planning/events/events_files.py]: not sure we need this, probably same as `_created` on the item

File: server/planning/events/events_files.py (hunk 1 above)
Comment (ride90) [reply server/planning/events/events_files.py]: yes, it is actually an alias for filename. Will drop it.

File: server/planning/events/events_files.py (hunk 2 above)
Comment (ride90) [reply server/planning/events/events_files.py]: yep, `AmazonObjectWrapper` does not have it. will drop it.

File: server/planning/events/events_files.py [hunk 4]
```diff
@@ -45,7 +46,13 @@ class EventsFilesService(superdesk.Service):
     def on_create(self, docs):
         for doc in docs:
             # save the media id to retrieve the file later
-            doc['filemeta'] = {'media_id': doc['media']}
+            _file = app.media.get(doc['media'])
+            doc['filemeta'] = {
+                'media_id': doc['media'],
```
Comment (petrjasek) [inline server/planning/events/events_files.py]: personally would drop this one too, it's there already as media

Comment (ride90) [reply server/planning/events/events_files.py]: I thought about it, but not sure, it might break something. Let's try to drop it.

**Pull request merged**